### ISSUE TITLE
panic: qtls.ClientSessionState not compatible with tls.ClientSessionState in go1.15

### ISSUE THREAD

**Issue by passionofvc** [ipfs/ipfs-cluster#1223]
ipfs-cluster-service run in panic with go 1.15.

```
$ go version
go version go1.15 linux/amd64

```
git clone https://github.com/ipfs/ipfs-cluster.git $GOPATH/src/github.com/ipfs/ipfs-cluster
cd $GOPATH/src/github.com/ipfs/ipfs-cluster
make install
```

$ ls  -ltr $GOPATH/bin/ipfs-cluster-*
-rwxrwxr-x. 1 passionofvc passionofvc 41547980  8月 27 19:20 /home/passionofvc/go/bin/ipfs-cluster-service
-rwxrwxr-x. 1 passionofvc passionofvc 21759901  8月 27 19:20 /home/passionofvc/go/bin/ipfs-cluster-ctl
-rwxrwxr-x. 1 passionofvc passionofvc 40594169  8月 27 19:20 /home/passionofvc/go/bin/ipfs-cluster-follow
```

```

$ ipfs-cluster-service version
panic: qtls.ClientSessionState not compatible with tls.ClientSessionState

goroutine 1 [running]:
github.com/lucas-clemente/quic-go/internal/handshake.init.2()
        pkg/mod/github.com/lucas-clemente/quic-go@v0.15.7/internal/handshake/unsafe.go:26 +0x205
```

**Comment by passionofvc** [comment]
#https://github.com/ipfs/go-ipfs/issues/7597

**Comment by hsanjuan** [comment]
Fixed in #1225 actually.
